_maps/configs/cascade.json

```
@@ -32,5 +32,4 @@
 		}
 
 	},
```

```suggestion
	}
```
the extra comma is causing an error